docs: explained how to see dispatched actions (#2281)
ngrx/platform: projects/ngrx.io/content/guide/store/testing.md

<div class="alert is-helpful">

> **Note:** All dispatched actions don't affect the state, but you can see them in the `Actions` stream. You can subscribe to it using `.scannedActions$.subscribe(...)` on the MockStore instance.

Reviewer: ```suggestion
**Note:** All dispatched actions don't affect the state, but you can see them in the `Actions` stream. You can subscribe to the stream with the `scannedActions$` property on the `MockStore` instance, for example `store.scannedActions$.subscribe(action => console.log(action))`.
```

Reviewer: What about the above?
